components/molecule/selectField/src/index.js

```diff
@@ -0,0 +1,80 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+
+import MoleculeField from '@s-ui/react-molecule-field'
+import MoleculeSelect from '@s-ui/react-molecule-select'
+
+const hasErrors = (success, error) => {
+  if (error) return true
+  if (success) return false
+}
+
+class MoleculeSelectField extends Component {
+  refSelect = React.createRef()
+  id = this.props.id || `MoleculeSelect-${+new Date()}`
```

Comment:
looks like we are not automagically settings ids in other components so I would remove this feature and add `isRequired` to the id propType in order to keep consistency across components. 

Also, developers are kind of forced to control de id in order to make it work with Formik or a custom form manager, so please remove it and just make it required

Reply:
Updated as `isRequired` prop